csv2sql.drush.inc

```diff
@@ -38,6 +44,9 @@ function csv2sql_drush_command() {
  */
 function drush_csv2sql($csv_path) {
   $prefix = drush_get_option('prefix', '_raw');
+  $delimiter = drush_get_option('delimiter', ',');
+  $enclosure = drush_get_option('enclosure', '"');
+  $escape = drush_get_option('escape', '\\');
```

Comment:
The default value we passed before was `null`, what's the effect with the new default value ?